assets/scss/components/_icon.scss

```diff
@@ -39,7 +33,9 @@
 	// (ensures the entire element is clickable, not just the drawn bits
 	position: relative;
 
-	use { pointer-events: none; }
+	use {
```

Comment:
I thought a single property was ok on one line?
Maybe I misunderstood the lint rule for this. Enforcing line breaking always is fine, too.

Reply:
We configured the rule to allow this, but it looks like stylelint autofixed this anyway. I'm fine with either